pwa/src/templates/templateParts/header/HeaderTemplate.tsx

```diff
@@ -122,13 +132,22 @@ export const HeaderTemplate: React.FC<HeaderTemplateProps> = ({ layoutClassName
       },
       icon: <GitHubLogo />,
     },
-    {
-      label: "Login",
-      handleClick: () => {
-        navigate("/login");
-      },
-      icon: <FontAwesomeIcon icon={faCircleUser} />,
-    },
+
+    isLoggedIn()
+      ? {
+          label: "Logout",
+          handleClick: () => {
+            navigate("/logout");
+          },
+          icon: <FontAwesomeIcon icon={faCircleUser} />,
+        }
+      : {
+          label: "Login",
+          handleClick: () => {
+            navigate("/login");
+          },
+          icon: <FontAwesomeIcon icon={faCircleUser} />,
+        },
   ];
   const {
```

Comment:
I think this location variable should be set at the top of this file, for consistency.